terraform/internal/rancher/network/main.tf

```
@@ -91,25 +91,55 @@ locals {
   flannel_rules = yamldecode(file("${path.module}/files/flannel.yaml"))["rules"]
 
   # Open Ports
-  open_port_rules = flatten([
-    for port in concat(var.open_ports) : [
+  open_port_rules = [
+    for port in var.open_ports : [
```

```suggestion
    for port in var.open_ports :
```
(and remove closing bracket)